pkg/infra/log/file.go

```diff
@@ -193,12 +193,12 @@ func (w *FileLogWriter) DoRotate() error {
 		// close fd before rename
 		// Rename the file to its newfound home
 		if err = os.Rename(w.Filename, fname); err != nil {
-			return fmt.Errorf("Rotate: %s", err)
+			return fmt.Errorf("rotate: %s", err)
 		}
 
 		// re-start logger
 		if err = w.StartLogger(); err != nil {
-			return fmt.Errorf("Rotate StartLogger: %s", err)
+			return fmt.Errorf("rotate StartLogger: %s", err)
```

Comment:
Same as above, do we want the error be wrapped here?

Reply:
@papagian See my reply above, I'm rather agnostic in the scope of this PR as to whether we want to do it or not.